### ISSUE TITLE
p7zip is not available on arm/v6

### ISSUE THREAD

**Issue by ttionya** [ttionya/vaultwarden-backup#68]
Rclone 1.59.0 is built on Alpine 3.16, the tool's dependency `p7zip` cannot be found on the Alpine 3.16 `arm/v6` platform.

I have created an issue jinfeihan57/p7zip#187 under the p7zip project and waiting for it to be fixed. In the meantime I will evaluate the possibility of using alternative tools.

See the [GitHub Actions](https://github.com/ttionya/vaultwarden-backup/runs/7275949769?check_suite_focus=true) log for errors.

```shell
#17 0.339 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/armhf/APKINDEX.tar.gz
#17 1.256 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/armhf/APKINDEX.tar.gz
#17 2.542 ERROR: unable to select packages:
#17 2.654   p7zip (no such package):
#17 2.654     required by: world[p7zip]
```

**Comment by ttionya** [comment]
https://gitlab.alpinelinux.org/alpine/aports/-/issues/14008

**Comment by ttionya** [comment]
Starting with `p7zip` 17.04, it is built on p7zip's fork [jinfeihan57/p7zip](https://github.com/jinfeihan57/p7zip), while the previous version (16.02) was built based on `p7zip` on [sourceforge](https://sourceforge.net/projects/p7zip/files/).

But `p7zip` 17.04 couldn't pass the test on the `armhf` platform, so the maintainer removed `armhf` platform support. You can see pipeline at https://gitlab.alpinelinux.org/alpine/aports/-/jobs/767452. And Alpine 3.16 was released after the removal of `armhf` platform, so it doesn't have the `p7zip` package.

I could build a separate image for `arm/v6` via the specific Dockerfile, but its tag would not be consistent with that of other platforms, so I did not use this method for ease of use.

Now I use multistage build, `arm/v6` based on `alpine:3.15`, and install the `rclone` package, but it's only version 1.57.0, other platforms are the latest 1.59.0.

**Comment by ttionya** [comment]
This issue will not be closed until `p7zip` can be used normally on all platforms.

**Comment by ttionya** [comment]
I found a new 7z package in `alpine:3.17` called [`7zip`](https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/7zip/APKBUILD). it is only available in `alpine:3.17`.

Now that rclone is built on `alpine:3.17`, maybe we can replace `p7zip` with `7zip`.
